api/catalog/api/admin.py

```diff
@@ -10,20 +11,20 @@
 )
 
 
-@admin.register(ImageReport)
-class ImageReportAdmin(admin.ModelAdmin):
-    list_display = ("reason", "status", "image_url", "description", "created_at")
+class AbstractMediaReportAdmin(admin.ModelAdmin):
     list_filter = ("status", "reason")
     list_display_links = ("status",)
     search_fields = ("description", "identifier")
     actions = None
 
+    class Meta:
+        abstract = True
```

Comment:
I don't think `ModelAdmin` classes need `abstract = True` in their meta. Removing this seems to have no effect on the outcome.

Reply:
Agreed. The MediaReport model itself is abstract but it's not needed in the model admin, and I don't imagine we'll have any use for it later. Removed :)